cibuildwheel/linux.py

```diff
@@ -6,14 +6,21 @@
 import uuid
 from collections import namedtuple
 
+from typing import Callable, Dict, List, Optional
+
+from .environment import (
+    ParsedEnvironment,
+)
```

Comment:
single line?

```python
from .environment import ParsedEnvironment,
```

Reply:
Maybe, yes. I guess I copied this, and removed an unused import afterwards.

Reply:
Isn't it nicer in style to have both imports the same?

```python
from .environment import (
    ParsedEnvironment,
)
from .util import (
    download,
    get_build_verbosity_extra_flags,
    prepare_command,
)
```

Reply:
and line above?
```python
 from typing import Callable, Dict, List, Optional
```

Reply:
Yes. this dicusssion is not strongly connected with PR clue.

Reply:
Nono, it's a valid point; I appreciate the comment! :-)
But I don't really know what's the best way to solve it.